### PR TITLE
DBZ-4839: update readme

### PR DESCRIPTION
**Pull request by ahmedjami**
https://issues.redhat.com/browse/DBZ-4839

With this new PR related to DBZ-4839, we will update the readme to notice that:
DBZ will be compatible Cassandra 4.x from 1.9
Building the project does not require the main DBZ repo
We no longer limited to java-8, the new version works with java-11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,24 +18,20 @@ You are encouraged to explore this connector and test it, but it is not recommen
...
+## Cassandra compatibility
 
-Building this connector first requires the main [debezium](https://github.com/debezium/debezium) code repository to be built locally using `mvn clean install`.
+The following table list supported version of Apache Cassandra:
 
-In order to build the Cassandra connector you'll need JDK 8 because Cassandra 3.x
-doesn't support Java versions above Java 8. That also means dependencies like
-`debezium-core` have to be built as Java 8 bytecode version 52.0 as well,
-either by compiling it with Java 8 or specifying Java 8 bytecode generation
-on newer versions of Java.
```
Comment (gunnarmorling) [inline README.md:29]: I think this is still true? No need to remove it?

**Review by gunnarmorling — COMMENTED** [review]
One comment inline, @ahmedjami!

**Comment by gunnarmorling** [comment]
> Building the project does not require the main DBZ repo

There definitely are dependencies on classes from _debezium-core_. Are you saying it doesn't have to be built locally because snapshot builds can be obtained from oss.sonaytype.org? If so, that's true, but building core locally is required when it comes to any cross-repo changes, like new or changed API/SPI contracts in core which are used in the Cassandra connector.

> We no longer limited to java-8, the new version works with java-11

That's not true for the Cassandra 3.x connector, is it? It would only be true once we test with Cassandra 3.x in a container.

**Comment by ahmedjami** [comment]
> That's not true for the Cassandra 3.x connector, is it? It would only be true once we test with Cassandra 3.x in a container.

Indeed, I think that I submit my changes very quickly after we have validated the support for Cassandra 4.X ^^
I will review my PR to just mention that we have now the support of the 2 versions of Cassandra and we will discuss support for Java, building.... in other topics :)

**Comment by Naros** [comment]
Hi @ahmedjami where do we stand on this PR?

**Comment by ahmedjami** [comment]
Hi @Naros 
Sorry for the delay...
For "building section", I think the simple way is to refer to the main repository so if we will have a new changes related to the java version or others, this repo will follow the changes mentionned in the main repo.

Here is what we will have in Cassandra readme:

> ## Building the Debezium Cassandra connector
> 
> Please follow the steps described in the main repository to build the connector: [building-debezium](https://github.com/debezium/debezium#building-debezium)
> 

WDYT ?

**Comment by Naros** [comment]
I think that's fine @ahmedjami, could you just do me a favor and remove the merge commit and we should be good.

**Comment by ahmedjami** [comment]
Hi @Naros 
I'm not be able to see the merge request that appears in my PR... :S 
Here is what I'm trying to do to remove it :+1: 

```
$ git status
On branch DBZ--4839
Your branch is up to date with 'origin/DBZ--4839'.

nothing to commit, working tree clean
```
```
git rebase -i HEAD~5

pick 54cea92 [maven-release-plugin] prepare release v1.9.0.Beta1
pick b745bf4 [maven-release-plugin] prepare for next development iteration
pick fa13b1b [release] New parent 1.9.0-SNAPSHOT for development
pick e4c1bfd DBZ-4839: update readme
pick aeb7110 DBZ-4839 Update README to mention the support of Cassandra-4


```
As you can see, the merge commit does not appear in the list of commits...
Do you have any idea please ?
Thanks

**Comment by ahmedjami** [comment]
I'll send a new PR based on last changes on main

**Pull request closed without merging**